packages/toggle/README.md

```diff
@@ -23,49 +23,56 @@ npm install @leafygreen-ui/toggle
-| ...         | native `input` attributes              | Any other props will be spread on the root `input` element                                                         |                           |
+| Prop        | Type                               | Description                                                                                                                                                                                                     | Default     |
+| ----------- | ---------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| `darkMode`  | `boolean`                          | Determines if the Toggle will render the dark mode styles.                                                                                                                                                      | `false`     |
+| `size`      | `'default'`, `'small'`, `'xsmall'` | Sets the size of the toggle.                                                                                                                                                                                    | `'default'` |
+| `checked`   | `boolean`                          | Set's the checked state of the Toggle.                                                                                                                                                                          |             |
+| `disabled`  | `boolean`                          | Disables the Toggle.                                                                                                                                                                                            | `false`     |
+| `onClick`   | `(MouseEvent) => void`             | The event handler function for the 'onchange' event. Receives the associated `event` object as the first argument.                                                                                              |             |
+| `onChange`  | `(checked, MouseEvent) => void`    | `onChange` fires when the internally-managed `checked` state of the component is updated. Receives the updated checked state of the toggle as its first argument, and the associated mouse event as the second. |             |
+| `className` | `string`                           | Adds a className to the outermost element.                                                                                                                                                                      | `''`        |
+| ...         | HTML `button` attributes           | Any natively-supported HTML button attributes will be applied to the `button` element                                                                                                                           |             |
```

Comment:
We have a convention for describing rest props now, check out banner's readme

Reply:
I was looking to improve clarity, especially if a developer using our components is unfamiliar with the term "spread". I've met developers at the company who have never used the syntax/don't know what it means, but they use our components.

"native" -> "HTML"
> This is a bit more specific and clear IMO

"Any other props will be spread on the root `div` element" ->" Any natively-supported HTML button attributes will be applied to the `button` element"
> This does a few things. First, the button isn't the root element, so that would be inaccurate. Second, by specifying button attributes it makes the expectations for what you can pass more clear as it mirrors more closely what React documents. Saying  "props" is only relevant if offering an `as` prop in a component IMO.